### PR TITLE
Docs for the upcoming release

### PR DESCRIPTION
**Pull request by kbevers**
Essentially a duplicate of https://github.com/OSGeo/proj.4/pull/563, this time on a branch in the project repository. This way it is easier for others to contribute as well. The overall structure of the docs are in place now, so feel free to correct existing text and add additional text.

Closes https://github.com/OSGeo/proj.4/issues/442
Closes https://github.com/OSGeo/proj.4/issues/454

## Want to contribute to the version 5.0.0 docs?

Awesome! It's quite simple:

* Clone the PROJ.4 repo
* Check out the branch `docs-release-4.10.0` (it was created before we decided on v. 5)
* Document something
* If  you have commit access to the main repo, you can push your changes directly to this branch, otherwise you will have to make a pull request targeting the `docs-release-4.10.0` branch. Please add `[skip ci]` to your commit messages (otherwise proj4.org will be updated with the v. 5.0.0 docs...).
* Feel good about yourself for giving some much needed love to the PROJ.4 docs!

The reason the docs for 5.0.0 are maintained in a separate branch is that changes to the docs in the master branch get's pushed to proj4.org as soon as they land in master. Un-released features should not be documented on the website.

## Readable version of the new docs: https://kbevers.github.io/ 

I try to keep this updated when new changes are made.

### CODE REVIEW

**Comment by kbevers** [comment]
Remember that a working copy of the docs can be seen at https://kbevers.github.io/

File: docs/source/usage/apps/cct.rst [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+*Coordinate Transformations*, which are coordinate operations where
+input and output datums differ (e.g. change of reference frame).
+
+``cct``, however, also refers to Carl Christian Tscherning (1942--2014),
+professor of Geodesy at the University of Copenhagen, mentor and advisor
+for a generation of Danish geodesists, colleague and collaborator for
+two generations of global geodesists, Secretary General for the
+International Association of Geodesy, IAG (1995--2007), fellow of the
+Amercan Geophysical Union (1991), recipient of the IAG Levallois Medal
```
Comment (mwtoews) [inline docs/source/usage/apps/cct.rst:36]: s/Amercan/American/

File: docs/source/usage/apps/cct.rst [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+(2007), the European Geosciences Union Vening Meinesz Medal (2008), and
+of numerous other honours.
+
+cct, or Christian, as he was known to most of us, was recognized for his
+good mood, his sharp wit, his tireless work, and his great commitment to
+the development of geodesy -- both through his scientific contributions,
+comprising more than 250 publications, and by his mentoring and teaching
+of the next generations of geodesists.
+
+As Christian was an avid Fortran programmer, and a keen Unix connoiseur,
```
Comment (mwtoews) [inline docs/source/usage/apps/cct.rst:46]: s/connoiseur/connoisseur/

**Comment by mwtoews** [comment]
Some of the typos fixed in #815 should to be brought in here too.

**Comment by kbevers** [comment]
> Some of the typos fixed in #815 should to be brought in here too.

 I have merged those in, so they should be included already. Except maybe one or two due to a minor merge conflict.

**Pull request merged**